Integrations/PhishLabsIOC_EIR/PhishLabsIOC_EIR.py

```diff
@@ -0,0 +1,432 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+''' IMPORTS '''
+from typing import Dict, Tuple, Union, Optional, List, Any, AnyStr
+import urllib3
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+"""GLOBALS/PARAMS
+Attributes:
+    INTEGRATION_NAME:
+        Name of the integration as shown in the integration UI, for example: Microsoft Graph User.
+
+    INTEGRATION_COMMAND_NAME:
+        Command names should be written in all lower-case letters,
+        and each word separated with a hyphen, for example: msgraph-user.
+
+    INTEGRATION_CONTEXT_NAME:
+        Context output names should be written in camel case, for example: MSGraphUser.
+"""
+INTEGRATION_NAME = 'PhishLabs IOC - EIR'
+INTEGRATION_COMMAND_NAME = 'phishlabs-ioc-eir'
+INTEGRATION_CONTEXT_NAME = 'PhishLabsIOC'
+
+
+class Client(BaseClient):
+    def test_module(self) -> Dict:
+        """Performs basic GET request to check if the API is reachable and authentication is successful.
+
+        Returns:
+            Response json
+        """
+        return self.phishlabs_ioc_eir_get_incident_by_id(incident_id='INC0660932')
+
+    def phishlabs_ioc_eir_get_incidents(self, limit=25, offset=0, **kwargs: dict) -> Dict:
+        """Query the specified kwargs with default parameters if not defined
+
+        Args:
+            offset: pagination parameter offset from the beginning of the page
+            limit: pagination parameter limit of incidents on a page 0..50
+            kwargs: parameters for filtering incidents
+
+        Returns:
+            Response JSON as dictionary
+        """
+        suffix = "/incidents/EIR"
+        return self._http_request('GET',
+                                  url_suffix=suffix,
+                                  params={**kwargs,
+                                          'offset': offset,
+                                          'limit': limit})
+
+    def phishlabs_ioc_eir_get_incident_by_id(self, **kwargs) -> Dict:
```

Comment:
no actual reason to use kwargs here.
you have only one acceptable argument and it is mandatory